prefs.js

```diff
@@ -88,14 +83,24 @@ function buildPrefsWidget(){
 		visibleFieldsBox.append(secondFieldComboBox,true,true,0);
 		visibleFieldsBox.append(lastFieldComboBox,true,true,0);
 	}
+	visibleFieldsBox.margin_start = 30; //include margin on left to align with rest of widgets
```

Comment:
proposed to include left padding to increase widget width so it drives the window with and ends up aligned on the right